src/main/java/com/icthh/xm/tmf/ms/activation/domain/spec/SagaTaskSpec.java

```diff
@@ -53,4 +57,18 @@ public void applyAsDefaultTransactionConfig(SagaTransactionSpec tx) {
             setter.accept(value);
         }
     }
+
+    @Override
+    public SagaTaskSpec clone() {
+        SagaTaskSpec cloned = null;
+        try {
+            cloned = (SagaTaskSpec) super.clone();
+        } catch (CloneNotSupportedException e) {
```

Comment:
why we are hiding Exception?

Reply:
See final version - try|catch removed according to @sergeysenja1992 comment